### DISCUSSION TITLE
Possibility of changing the pinned sites on the New Tab Page

### DISCUSSION THREAD

**Discussion by Zagrthos** [mozilla/policy-templates#1030] [Q&A] [answered]
Is it possible to change the pinned sites on the new tab page to my own?

Tried it with the Preferences policy like this:

	"browser.newtabpage.pinned": {
		"Value": [{"url":"https://test1.de/","label":"Test1"},{"url":"https://test2.de/","label":"Test2"}],
        "Status": "locked"
	}

**Comment by mkaply** [comment, marked as answer]
Sorry for the delay in getting back.

Because this is actually a string in Firefox preferences, you can't put it as actual JSON for the value.

This will work (ugly, I know):

```
{
	"browser.newtabpage.pinned": {
		"Value": "[{\"url\": \"https://test1.de/\",\"label\": \"Test1\"},{\"url\": \"https://test2.de\",\"label\": \"Test2\"}]",
		"Status": "locked"
	}
}
```

**Comment by mkaply** [comment]
(I tried using single quotes, but it didn't work. This is the only way)

**Comment by Zagrthos** [comment]
No worries for the long time without an answer!
Works now like a charm. Thank you! 👍
